src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -57,10 +57,11 @@ foreach ($defaultEnvs as $envName => $envValue) {
     }
 }
 
+$isWindows = '\\' === DIRECTORY_SEPARATOR;
 $COMPOSER = file_exists($COMPOSER = $oldPwd.'/composer.phar')
-    || ($COMPOSER = rtrim('\\' === DIRECTORY_SEPARATOR ? preg_replace('/[\r\n].*/', '', `where.exe composer.phar`) : `which composer.phar 2> /dev/null`))
-    || ($COMPOSER = rtrim('\\' === DIRECTORY_SEPARATOR ? preg_replace('/[\r\n].*/', '', `where.exe composer`) : `which composer 2> /dev/null`))
-    ? $PHP.' '.escapeshellarg($COMPOSER)
+    || ($COMPOSER = rtrim($isWindows ? preg_replace('/[\r\n].*/', '', `where.exe composer.phar`) : `which composer.phar 2> /dev/null`))
+    || ($COMPOSER = rtrim($isWindows ? preg_replace('/[\r\n].*/', '', `where.exe composer`) : `which composer 2> /dev/null`))
+    ? (!$isWindows && (strpos(`file $COMPOSER 2> /dev/null`, 'binary data') === false) ? '' : $PHP).' '.escapeshellarg($COMPOSER)
```

Comment:
instead, I'd suggest checking if the file starts by `#!/usr/bin/env php`, both on windows and on linux

Reply:
wrapped does not have to be a php file.
Example `$ cat phpenv which composer`

```bash
$ cat `phpenv which composer`

#!/usr/bin/env bash
# Summary: phpenv composer plugin

set -e

prefix=$(phpenv prefix)
...
```

Reply:
There could be at least 2 different shebangs:
```
#!interpreter [optional-arg]

#!/bin/sh (bash or any other shell)
#!/usr/bin/env python (php bash sh and etc.)
``` 

Need to avoid appending php-executable for at least following cases:
```
#!/usr/bin/env php
#!<anything_path_to>php
#!/bin/sh
#!/bin/bash
```

so I propose to not to append php-executable if any shebang `#!`  is detected.

Reply:
Works for me.

Reply:
updated PR.

Reply:
Updated PR